Reference/Events/EditorModel-Events/Customizing-the-links-box-v9.md

```diff
@@ -0,0 +1,26 @@
+# Customizing the "Links" box
+
+For a content item, Umbraco will show a **Links** box within the **Info** content app. By default, this box will show one or more links to content item.
+
+![image](Images/properties-info-app.png)
+
+With the `SendingContentNotification` event, we can manipulate the links in the `Urls` property - e.g. replace it with some custom links (although a URL provider would be more suitable):
+
+```C#
+public void Handle(SendingContentNotification notification)
+{
+    notification.Content.Urls = new[]
+    {
+        new UrlInfo($"/products/?id={notification.Content.Id}", true, CultureInfo.CurrentCulture.Name)
+    };
+}
+```
```

Comment:
No the semicolon is in the right place, it's because it's a collection initializer that has a bit of a weird syntax 😄 

The curly brackets are more like a parenthesis, in this case, it becomes a bit more apparent when you add more than one item, then it would look like: 

```C#
notification.Content.Urls = new[]
{
    new UrlInfo($"/products/?id={notification.Content.Id}", true, CultureInfo.CurrentCulture.Name),
    new UrlInfo($"/SomeOtherLink/?id={notification.Content.Id}", true, CultureInfo.CurrentCulture.Name)
};
```
If it makes sense? 😄

Reply:
Aah! Well, I learned something new then 💪 
Thanks for explaining!